mmdet/datasets/coco.py

```diff
@@ -60,6 +66,26 @@ def _filter_imgs(self, min_size=32):
                 valid_inds.append(i)
         return valid_inds
 
+    def get_subset_by_classes(self, class_ids=None):
+        '''Get img ids that contain any category in class_ids.
+
+        Different from the coco.getImgIds, this function returns the id if
+        the img contains one of the cat rather than all.
```

Comment:
Different from the `coco.getImgIds()`, this function returns the id if the img contains one of the categories rather than all.